src/controllers/products.ts

```diff
@@ -60,18 +60,30 @@ export class ProductController {
     next: NextFunction
   ): Promise<void> {
     try {
-      const { limit, page, sort, active } = req.query
-      const filter: ProductsListFilter = { account: req.decoded?.user.account }
+      const { limit, page, sort, active, id, toPaginate = 'true' } = req.query
+
+      const filter: ProductsListFilter = {
+        account: req.decoded?.user.account,
+        id,
```

Comment:
Se nao for enviado o id na query string, nao vai dar problema?

Se der seria interessente fazer:
`if (id) filter.id = id`